src/views/ItemAdmin/ItemAdminObjective.vue

```diff
@@ -38,6 +38,15 @@
         @select="changedPeriod = true"
       />
 
+      <form-component
```

Comment:
Hm, jeg er usikker på om vi burde vise denne for mål som allerede ligger i en periode, for de har jo start- og sluttdato gitt av perioden. Kanskje bare tillate å sette start og slutt på nye mål som lages utenfor en periode?